src/index.scss

```diff
@@ -1,5 +1,8 @@
 @mixin type($background-color) {
   background-color: $background-color;
+  border-radius: 3px;
+  font-size: 13px;
+  padding: 8px 21px;
```

Comment:
I see what you want to do - but adding them to something named 'background-color' seems misleading. Maybe we can define another mixing type, like base-sizing or something?